### PR TITLE
Add DEC 2024 rewards

### PR DESCRIPTION
**Pull request by mohammadranjbarz**
related to https://github.com/Giveth/giveth-dapps-v2/issues/4869

### CODE REVIEW

File: deployments/notifyRewardAmount/optimism/givpower_distribute_extended_dec_2024.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+const hre = require("hardhat");
+const { sendReportEmail } = require("../../mailService/mailService");
+const { ethers } = hre;
+
+const pools = [
+    {
+        address: "0x301C739CF6bfb6B47A74878BdEB13f92F13Ae5E7",
+
+        // https://github.com/Giveth/giveth-dapps-v2/issues/4434
+        amount: "3825000",
```
Comment (aminlatifi) [inline deployments/notifyRewardAmount/optimism/givpower_distribute_extended_dec_2024.js:11]: According to https://github.com/Giveth/giveth-dapps-v2/issues/4869
It should be: 3,584,250

File: deployments/notifyRewardAmount/xDAI/givpower_distribute_extended_dec_2024.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+const hre = require("hardhat");
+const { sendReportEmail } = require("../../mailService/mailService");
+const { ethers } = hre;
+
+const pools = [
+    {
+        address: "0xD93d3bDBa18ebcB3317a57119ea44ed2Cf41C2F2",
+
+        // https://github.com/Giveth/giveth-dapps-v2/issues/4434
+        amount: "2550000",
```
Comment (aminlatifi) [inline deployments/notifyRewardAmount/xDAI/givpower_distribute_extended_dec_2024.js:11]: Likewise, it should be 2,389,500

File: deployments/notifyRewardAmount/zkevm/givpower_distribute_extended_dec_2024.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+const hre = require("hardhat");
+const { sendReportEmail } = require("../../mailService/mailService");
+const { ethers } = hre;
+
+const pools = [
+    {
+        address: "0xc790f82bf6f8709aa4a56dc11afad7af7c2a9867",
+
+        // https://github.com/Giveth/giveth-dapps-v2/issues/4434
+        amount: "2125000",
```
Comment (aminlatifi) [inline deployments/notifyRewardAmount/zkevm/givpower_distribute_extended_dec_2024.js:11]: Likewise, it should be 1,991,250

**Review by aminlatifi — CHANGES_REQUESTED** [review]

File: deployments/notifyRewardAmount/optimism/givpower_distribute_extended_dec_2024.js (hunk 1 above)
Comment (mohammadranjbarz) [reply deployments/notifyRewardAmount/optimism/givpower_distribute_extended_dec_2024.js]: Done.

File: deployments/notifyRewardAmount/xDAI/givpower_distribute_extended_dec_2024.js (hunk 2 above)
Comment (mohammadranjbarz) [reply deployments/notifyRewardAmount/xDAI/givpower_distribute_extended_dec_2024.js]: Done.

File: deployments/notifyRewardAmount/zkevm/givpower_distribute_extended_dec_2024.js (hunk 3 above)
Comment (mohammadranjbarz) [reply deployments/notifyRewardAmount/zkevm/givpower_distribute_extended_dec_2024.js]: Done.

**Comment by mohammadranjbarz** [comment]
Thanks @aminlatifi I pushed the changes

**Review by aminlatifi — COMMENTED** [review]
LGTM, thanks @mohammadranjbarz

**Review by aminlatifi — APPROVED** [review]

**Comment by mohammadranjbarz** [comment]
@aminlatifi @laurenluz I merged/ deployed and modified the scripts on the server to execute these new scripts

**Comment by aminlatifi** [comment]
Thanks @mohammadranjbarz 
I checked it on the server, looks good. But let keep our eyes open on next notify reward time

**Pull request merged**